### PR TITLE
fix: dynamic operator wallets in create program form

### LINKED ISSUE #176: Create Program Page operators fields overwrite each other
**Overview**
Currently, when I am on the Create Program page, when I try to add multiple operators, whatever field is being typed into overwrites the value in the other fields when trying to save the Program.

**Steps To Reproduce**
1. Go to `/program/create`
2. Enter a name
3. Add multiple operators with different addresses
4. Click `Save`

**Observed  behavior**
All operator fields change to the last value entered for an operator address

**Expected behavior**
The operator values remain as entered

**Additional context**
It looks like there was already some code added to address this, but no story in the backlog references this issue.

https://github.com/gitcoinco/grants-round/blob/dded4eda6075c08c19b8b81f3d55a999dbe10a01/packages/round-manager/src/features/program/CreateProgramPage.tsx#L147-L148

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This fixes the bug in program creation form where operator wallets overwrite each other

##### Refers/Fixes

closes #176

### CODE REVIEW

File: packages/round-manager/src/features/program/CreateProgramPage.tsx [hunk 1]
```diff
@@ -78,74 +78,72 @@ export default function CreateProgram() {
...
-              <ul>
-                {fields.map((item, index) => {
-                  return (
+                <ul>
+                  {fields.map((item, index) => (
                     <li key={item.id} className="flex flex-rows">
                       <Input
-                        {...register(`operators.0.wallet`)}
-                        // {...register(`operators.${index}.wallet`)}
+                        {...register(`operators.${index}.wallet` as "operators.0.wallet")}
```
Comment (shavinac) [inline packages/round-manager/src/features/program/CreateProgramPage.tsx:146]: you can avoid this type assertion by updating the `FormData` type to
```
type FormData = {
  name: string;
  operators: { wallet: string }[];
}
```

before the `FormData` type actually only allows exactly 1 `{ wallet: string }` in the `operators` array

Comment (shavinac) [reply packages/round-manager/src/features/program/CreateProgramPage.tsx]: i pushed this change in commit [a088c91](https://github.com/gitcoinco/grants-round/pull/214/commits/a088c9166c93afaeabfcfc784f4859ebb183f354) here

**Review by pragmatriot — APPROVED** [review]
LGTM

**Comment by thelostone-mc** [comment]
could we add screeenshot/video as well please !
let's make this a habit within the team and ask for it while reviewing

**Pull request merged**